lib/alex_koin/slack_commands.ex

```diff
@@ -43,6 +64,14 @@ defmodule AlexKoin.SlackCommands do
     coin
   end
 
+  def transfer(from_wallet, to_wallet, amount, memo) do
+    # 1. get coins to transfer
+    coins = Coin |> Ecto.Query.where(wallet_id: ^from_wallet.id) |> Ecto.Query.limit(^amount) |> Repo.all
+
+    # 2. move the coins over to the other wallet
+    coins |> Enum.each(fn(c) -> transfer_coin(c, from_wallet, to_wallet, memo) end)
```

Comment:
Please don't pipe if there is only one. Enum.each(coins, fn ->) like syntax please.